File: birdseyelib/saveState.py
class SaveState:
    def __init__(self, client) -> None:
        self.client = client
        self.slot = 1

    def setSlot(self, slot):
        self.slot = slot
    
    def loadSlot(self, slot = None):
        if slot:
            self.slot = slot
        
        self.client._queue_request("LOADSLOT;" + str(self.slot) + "\n")

File: exttool/SaveState.cs
using System;
using System.Collections.Generic;
using System.Security.Cryptography.X509Certificates;
using BizHawk.Client.Common;

namespace BirdsEye {
    public class SaveState{


        private readonly Logging _log;
        public SaveState(Logging log){
            _log = log;
        }

        public void GetScreenshot(ApiContainer APIs, String path){
            APIs.EmuClient.Screenshot(path);
        }

        public bool LoadSlot(ApiContainer APIs, String slot){
            int saveSlot = Int32.Parse(slot); 
            return APIs.SaveState.LoadSlot(saveSlot);
        }
    }
}